### DISCUSSION TITLE
Multipule sites with one install

### DISCUSSION THREAD

**Discussion by James4645** [statamic/cms#4440] [Q&A]
Idk if this is possible but I manage multiple sites. Is there a way to manage multiple sites (each with their own license) from one statmic install so I only have to manage one install and have one login to manage each site? Sort of similar to a WordPress Multi-site setup?

**Comment by duncanmcclean** [comment]
Statamic has [multi-site](https://statamic.dev/multi-site#content) functionality, is that what you're looking for?

**Reply by James4645** [reply]
That's the thing, I have no clue. 

Because what little bit is said about it makes it sound like its designed to manage multiple translations of one site rather than multiple distinct websites.

And the fact that is does not appear to require multiple licenses makes me think no, that its not designed for that. 

I could probably do some fancy swapping at runtime in Laravel, for example connect it to a DB instead of flat files, but then dynamically swap out the database it connects to based on the URL or something?

The goal is I need multiple different websites but I want to maintain a single server with a single installation. The actual sites will be statically generated and uploaded to their own s3 bucket for actual live web traffic.

**Reply by duncanmcclean** [reply]
Oh okay, if it's completely different sites then you'd probably want different installs. If they're similar page structures & content, then maybe you could use multi-site and just switch around the templates,

**Reply by James4645** [reply]
That's the thing I don't want to have to maintain different installs. Idk that Statmic is really designed to work with one install though.

Kind of like with Wordpress how you can have one install but using multisite you get distinct separate websites that only share users for a common login. That's what I really want, but with Statamic.

**Reply by duncanmcclean** [reply]
I mean, you could give it a try and see how far you can get with multisite.

**Comment by jasonvarga** [comment]
If they're literally different websites, then they should be different installs with their own licenses.

The multisite feature is for managing translations, or for managing different parts of the same organization (e.g. a parent company and child brands, or a restaurant with different locations).

**Reply by James4645** [reply]
thats the kinda confusing part. These aren't different businesses or anything they are all different sites operated by my company. 

I just want an option to give each one its own license but manage it all as one installation.

**Reply by jasonvarga** [reply]
Statamic is not designed to be used that way.

**Reply by James4645** [reply]
Why though. Like seriously whats the difference? Seems like a pointless distinction.

**Reply by jackmcdade** [reply]
Complexity. It’s not an arbitrary thing to accomplish, making every feature contextually aware of completely separate sites. It’s not as simple as adding a site_id column to every table or something like that. 

What you COULD do — because this is a flat file platform — is build separate sites but share the same user files so you can log into all the sites, and then build a little addon that lets you switch BETWEEN your sites in the control panel (like a drop down in the nav bar or something) and then it would FEEL like one super site.

**Reply by James4645** [reply]
Is it though? Because you already have Multi-site you are just putting arbitrary limitations on its use.It seems the distinction is purely to force the use of multiple licenses which is fine, the purpose isn't to do everything with one license, it is to maintain a single install on a single server and not have to maintain an entire copy for each website.

**Reply by jackmcdade** [reply]
The multi-site feature is intended for translations and variations of a single site, not separate ones. There's a lot of blended content that goes on, they are not completely separate sites.
